geomesa-core/src/main/scala/org/locationtech/geomesa/core/iterators/IteratorTrigger.scala

```diff
@@ -94,25 +95,32 @@ object IteratorTrigger extends Logging {
    */
   def useIndexOnlyIterator(ecqlPredicate: Option[Filter],
                            query: Query,
-                           sourceSFT: SimpleFeatureType,
+                           sft: SimpleFeatureType,
                            indexedAttribute: Option[String] = None): Boolean = {
+    if (useDensityIterator(query: Query)) {
```

Comment:
what's up with the argument to useDenesityIterator... I feel like I'm reading this wrong or something